andes/cert-manager/cm-and-clusterissuer.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -exu
+
+kubectl create ns cert-manager
+helm repo add jetstack https://charts.jetstack.io
+kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.11/deploy/manifests/00-crds.yaml --validate=false
+helm install cert-manager -n cert-manager jetstack/cert-manager
+cat > secret.yaml << END
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-route53
+  namespace: cert-manager
+data:
+  aws_key: $(AWS_SECRET_KEY)
+
+END
+kubectl apply -f secret.yaml
+cat > letsencrypt.yaml << END
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt
+  namespace: cert-manager
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory 
+    privateKeySecretRef:
+      name: letsencrypt
+    email: $(EMAIL)
```

Comment:
Bash variable expansion is `${VAR}`; shell command expansion is `$(command)`.

This is a hard blocker as this script cannot work until variable expansion is corrected.